{"name": "test_login_error[item2]", "status": "passed", "parameters": [{"name": "item", "value": "{'username': '888888', 'pwd': '5466544', 'code': '123456', 'excepted': '抱歉，账号或密码错误！'}"}], "start": 1558343756979, "stop": 1558343758452, "uuid": "97ac7b98-5c91-4f4f-b657-7d49d41aa250", "historyId": "4d61c4440e2fe708de54238acd3e621d", "fullName": "TestCase.test_0_login.TestLogin#test_login_error[item2]", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_0_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "bj_jd-PC"}, {"name": "thread", "value": "6240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_0_login"}]}